### ISSUE TITLE
[SCREENREADER, KEYBOARD]: Selectable issue cards are not accessible

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#4907]
## Description

This issue was originally brought up with a Higher-Level Review a11y review ([ref](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3566#issuecomment-559214317)), but is an issue associated with all forms. cc @jenstrickland 

On a form page that allows users to choose from a list of issues, the card component (`SelectArrayItemsWidget`) does not give any guidance that it is interactive. The screenreader does not mention that the checkbox is available.

<img src="https://user-images.githubusercontent.com/57469/69751314-d21b9180-111c-11ea-8592-da4b9cd049d7.png" alt="Screenshot of contested issue card" />

## Acceptance Criteria

- [ ] Focus on the card should make the screenreader inform the user about the content and that there is a checkbox available for interaction.
- [ ] Keyboard navigation should focus on the card

## Steps to Recreate

1. Log in as user 218, then go to a form that contains a phone & email or mailing address review card, e.g. https://staging.va.gov/disability-benefits/apply/form-0996-higher-level-review/contested-issues
2. Turn on a screenreader.
3. Use the keyboard to <kbd>Tab</kbd> to the first contested issue card.
4. The screen reader doesn't notify the user that the checkbox is available for interaction.

## Possible Fixes (optional)

Currently each issue card is a `<dt>` element. Maybe changing this outer wrapper into a `<label>` (and removing the now inner label that only wraps the text content, not the checkbox) would fix this issue?

**Comment by jenstrickland** [comment]
@Mottie I'm trying to check on this but am unable to access the page. The link provided returned a 404, so I changed the url to https://staging.va.gov/disability-benefits/apply/form-0996-higher-level-review/introduction, then clicked Continue your application. The next page loads, and then disappears, revealing Sorry, we can't find that page. Is there something else I should do?

**Comment by Mottie** [comment]
Oh, sorry! The path isn't set up on the backend yet. You'll need to go to https://staging.va.gov/disability-benefits/apply/form-0996-higher-level-review/ first, then progress through the form until you get to that step.

**Comment by jenstrickland** [comment]
Here's the code currently displaying:
```HTML

<div class="schemaform-widget-wrapper">
<div>
<dt class="form-checkbox widget-outline">
<input type="checkbox" id="root_contestedIssues_0" name="root_contestedIssues_0" required="">
<label class="schemaform-label" for="root_contestedIssues_0">
<div class="widget-content">
<h4>Tinnitus</h4>
<span>Rinnging in the ears. More intese in right ear. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
<p>Current rating: <strong>10%</strong></p>
</div>
</label>
</dt>
<dd></dd>
</div>
<div>
<dt class="form-checkbox widget-outline">
<input type="checkbox" id="root_contestedIssues_1" name="root_contestedIssues_1" required="">
<label class="schemaform-label" for="root_contestedIssues_1">
<div class="widget-content">
<h4>Headaches</h4>
<span>Acute chronic head pain</span>
<p>Current rating: <strong>50%</strong></p>
</div>
</label>
</dt>
<dd></dd>
</div>
</div>

```

It would be preferable if the list of options read to a screen reader as a _list_ of terms and data that describes the term. Here's an example of what would work semantically, and should allow focus to the checkbox:
```HTML

<div class="schemaform-widget-wrapper">
<dl>
  <dt class="form-checkbox widget-outline">
    <input type="checkbox" id="root_contestedIssues_0" name="root_contestedIssues_0" required="">
    <label class="schemaform-label" for="root_contestedIssues_0">Tinnitus</label>
  </dt>
  <dd>
    <dfn>Description:</dfn> Ringing in the ears. More intense in right ear. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </dd>
  <dd>
    <dfn>Current rating:</dfn> 10%
  </dd>

  <dt class="form-checkbox widget-outline">
    <input type="checkbox" id="root_contestedIssues_1" name="root_contestedIssues_1" required="">
    <label class="schemaform-label" for="root_contestedIssues_1">Headaches</label>
  <dd><dfn>Description:</dfn> Acute chronic head pain</dd>
  <dd><dfn>Current rating:</dfn> 50%</dd>
</dl>
</div>

```

**Comment by Mottie** [comment]
Oh, yes, in my suggested fix, I proposed changing the `<dt>` into a `<label>` that way interacting with it would toggle the checkbox.

**Comment by Mottie** [comment]
OBE: the card HTML has changed significantly in the past year and has recently passed our accessibility review.
